src/rtext.c

```diff
@@ -1425,14 +1424,7 @@ int TextCopy(char *dst, const char *src)
 // REQUIRES: strcmp()
 bool TextIsEqual(const char *text1, const char *text2)
 {
-    bool result = false;
-
-    if ((text1 != NULL) && (text2 != NULL))
```

Comment:
Please do not remove this check.
`TextIsEqual(NULL, NULL)` or `TextIsEqual("abc", NULL)` or `TextIsEqual(NULL, "abc")` would cause a segfault.
Also, I believe that's the standard structure ([reference](https://github.com/raysan5/raylib/commit/b68d0850b1dee42d818e67d99f4affe1d890dba1#diff-39fea0e3daa23741e81daf967717f35c7842426aab26a0bc3f525d6a2f03e339R3767-R3777)).